doc/toolbox-sh.1.md

```diff
@@ -0,0 +1,21 @@
+% toolbox-sh(1)
+
+## NAME
+toolbox\-sh - Launch user configured shell inside container
```

Comment:
There are other use-cases for this: https://github.com/containers/toolbox/issues/988

It can have an impact on the name.  Although, since it's an internal implementation detail we can always change the name of the command later.